chassis/utils/chassis_module_set_admin_state.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Name: chassis_module_set_admin_state.py, version: 1.0
```

Comment:
Please add 

# Copyright (c) 2024 Nokia

Reply:
Fix it.

Reply:
Addressed.